tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -249,8 +248,24 @@ max-allowed-packet = 67_108_864
 # these as true in the production environment.
 # The execution order: Checksum -> Analyze
 [post-restore]
+<<<<<<< HEAD
 # Performs `ADMIN CHECKSUM TABLE <table>` for each table to verify data integrity.
 checksum = true
+=======
+# Specifies whether to perform `ADMIN CHECKSUM TABLE <table>` for each table to verify data integrity after importing.
+# The following options are available:
+# - "required" (default value): Perform admin checksum. If checksum fails, TiDB Lightning will exit with failure.
+# - "optional": Perform admin checksum. If checksum fails, TiDB Lightning will report a WARN log but ignore any error.
+# - "off": Do not perform checksum.
+# Note that since v4.0.8, the default value has changed from "true" to "required".
+# For backward compatibility, bool values "true" and "false" are also allowed for this field.
+# "true" is equivalent to "required" and "false" is equivalent to "off".
+checksum = "required"
+# Specifies whether to perform `ANALYZE TABLE <table>` for each table after checksum is done.
+# Options available for this field are the same as `post-restore`. However, the default value for this field is "optional".
+analyze = "optional"
+
+>>>>>>> 73f962dd7 (lightning: update config description of checksum and analyze (#7129))
```

Comment:
```suggestion

# Specifies whether to perform `ADMIN CHECKSUM TABLE <table>` for each table to verify data integrity after importing.
# The following options are available:
# - "required" (default value): Perform admin checksum. If checksum fails, TiDB Lightning will exit with failure.
# - "optional": Perform admin checksum. If checksum fails, TiDB Lightning will report a WARN log but ignore any error.
# - "off": Do not perform checksum.
# Note that since v4.0.8, the default value has changed from "true" to "required".
# For backward compatibility, bool values "true" and "false" are also allowed for this field.
# "true" is equivalent to "required" and "false" is equivalent to "off".
checksum = "required"
# Specifies whether to perform `ANALYZE TABLE <table>` for each table after checksum is done.
# Options available for this field are the same as `post-restore`. However, the default value for this field is "optional".
analyze = "optional"

```